geometric/engine.py

```diff
@@ -1,27 +1,19 @@
 """
 engine.py: Communicate with QM or MM software packages for energy/gradient info
-
```

Comment:
Your version has a number of blank lines removed. Could you please put them back?